addon/adapters/children.js

```diff
@@ -0,0 +1,11 @@
+import OsfAdapter from './osf-adapter';
+
+export default OsfAdapter.extend({
+    buildURL(_, __, snap, requestType) {
+        // Embed contributors
```

Comment:
How does this adapter embed contributors? It's not clear they're mentioned from the code below; a more descriptive comment might be helpful for some of the trickery below.